javascript/javascript2/week1/exercise01.js

```diff
@@ -0,0 +1,92 @@
+//Write a function that finds the shortest word of an array of words
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+const lengthOfArray = danishWords.length;
+function shortestWords(danishWords) {
+  for (let i = 0; i < lengthOfArray; i++) {
+    let arraysIndexLen = danishWords[i].length;
+    console.log(arraysIndexLen);
+    if (arraysIndexLen <= 1) {
```

Comment:
What will happen if there is shortest word with 2 char long, this condition fails. Better compare the one element in array with the another and keep the shortest one in on variable then compare it.